### ISSUE TITLE
User Quota (server issue or?)

### ISSUE THREAD

**Issue by zkenstein** [StudioMercury/digital-publishing-tools-for-wordpress#35]
Hi @StudioMercury ,

I tried to publish the article to DPS but I got the error like screenshot here: http://postimg.org/image/mnw0bpbk7/ 

Thanks,

**Comment by zkenstein** [comment]
Anyone faced this before?

**Comment by jerwarren** [comment]
@zkenstein make sure the user that you created the DPS api key with is explicitly added to your dps project and that the user has necessary api permissions.  That always gets me.

**Comment by zkenstein** [comment]
Hi @jerwarren thanks for the comments, yes the users already whitelisted but I still faced this issue: the error inside the body: Body
{"code":"ForbiddenException","message":"Article entity=/publication/980a3675-d047-493d-997c-22c189182006/article/hello-world-part;version=1450770780018 could not be published. Make sure articles have a thumbnail and article content."}

:(

Thx

**Comment by StudioMercury** [comment]
The error you’re getting is because the article doesn’t have a thumbnail or article file attached. In the WP plugin, open the article, attach a thumbnail. Then click on the cloud button and select “Update ALL” this will update your metadata, upload the thumbnail, and the article folio file. After that you can publish the article.

**Comment by zkenstein** [comment]
Hi @StudioMercury my article has a thumbnail but still not working (got same error).

When I tried to used inspect element I got this error: Failed to load resource: the server responded with a status of 403 (Forbidden)

Access forbidden, I don't have idea why this happened, the API user I used is same with my entitlement server 2015.

Updated: Has been fix, I will closed this. Very nice plugin.

Regards,
